src/components/PopupWithConfirmation.js

```diff
@@ -0,0 +1,21 @@
+import Popup from "./Popup.js";
+
+// Change name
+export default class PopupWithConfirmation extends Popup {
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.

Reply:
Thank you!